src/game/Game.ts

```diff
@@ -0,0 +1,128 @@
+import KeyListener from "./KeyListener";
+
+enum Tile {
+    FLOOR,
+    WALL,
+    UNDEFINED,
+}
+
+interface PlayerObject {
+    r: number;
+    c: number;
```

Comment:
Can we use more descriptive names for these properties? Took me a bit to assume it means `row` and `column`.